src/compiler/mir/ir.rs

```diff
@@ -463,6 +463,11 @@ impl BasicBlock {
         self.terminator = Some(term);
     }
 
+    /// Returns an [`Iter`] over the statements in this Basic Block
+    pub fn stm_iter(&self) -> Iter<Statement> {
+        self.statements.iter()
+    }
+
     /// Get the [`Statement`] at the given index
     pub fn get_stm(&self, idx: usize) -> &Statement {
         &self.statements[idx]
```

Comment:
Can this method be deleted?

Reply:
I think so, but will delete in the LLVM Transformer PR because that's where I created the function that returns an iterator over statements.